### ISSUE TITLE
[Bug] slow fine-tuning for wan_1.3b_vsa

### ISSUE THREAD

**Issue by clytze0216** [hao-ai-lab/FastVideo#605]
### Describe the bug

Thank you for your contribution to the fine - tuning vsa, when I was in fine-tuning FastVideo/scripts/finetune finetune_v1_VSA. Sh, I found that the average step_time around 6 s, And when I run FastVideo/scripts/finetune/finetune_v1 sh to full attention, time at about 1.5 s, according to the definition of vsa training speed should be the full attention of 6 times, Why is it so slow in actual operation?

### Reproduction

The following is the vsa fine-tuing script:

export WANDB_PROJECT=fastvideo-vsa

export TRITON_CACHE_DIR=/tmp/triton_cache

NUM_GPUS=4

export FASTVIDEO_ATTENTION_BACKEND=VIDEO_SPARSE_ATTN

CHECKPOINT_PATH="$DATA_DIR/outputs/wan_finetune_vsa/checkpoint-500"

# If you do not have 32 GPUs and to fit in memory, you can: 1. increase sp_size. 2. reduce num_latent_t

torchrun --nnodes 1 --nproc_per_node $NUM_GPUS \

--master_port=25368 \

FastVideo/fastvideo/v1/training/wan_training_pipeline.py \

--model_path Wan-AI/Wan2.1-T2V-1.3B-Diffusers \

--inference_mode False \

--pretrained_model_name_or_path Wan-AI/Wan2.1-T2V-1.3B-Diffusers \

--cache_dir "/home/ray/.cache" \

--data_path "$DATA_DIR" \

--validation_preprocessed_path "$VALIDATION_DIR" \

--train_batch_size 1 \

--num_latent_t 8 \

--sp_size 1 \

--tp_size 1 \

--num_gpus $NUM_GPUS \

--hsdp_replicate_dim $NUM_GPUS \

--hsdp-shard-dim 1 \

--train_sp_batch_size 1 \

--dataloader_num_workers 4 \

--gradient_accumulation_steps 8 \

--max_train_steps 50000 \

--learning_rate 1e-5 \

--mixed_precision "bf16" \

--checkpointing_steps 100 \

--validation_steps 500 \

--validation_sampling_steps "2,4,8,50" \

--log_validation \

--checkpoints_total_limit 3 \

--allow_tf32 \

--ema_start_step 0 \

--cfg 0.0 \

--output_dir "$DATA_DIR/outputs/wan_finetune_vsa" \

--tracker_project_name VSA_finetune \

--num_height 448 \

--num_width 832 \

--num_frames  61 \

--flow_shift 3 \

--validation_guidance_scale "5.0" \

--num_euler_timesteps 50 \

--master_weight_type "fp32" \

--dit_precision "fp32" \

--weight_decay 0.01 \

--max_grad_norm 1.0 \

--VSA_sparsity 0.9 \

--VSA_decay_rate 0.03 \

--VSA_decay_interval_steps 30 \

--resume_from_checkpoint "$CHECKPOINT_PATH"

### Environment

vsa

**Comment by BrianChen1129** [comment]
This might be because ```gradient_accumulation_steps``` is set to 8 in the VSA scripts.

**Comment by zhisbug** [comment]
Close as the issue seems to be addressed. Feel free to re-open.
